x-pack/legacy/plugins/monitoring/server/lib/metrics/apm/metrics.js

```diff
@@ -115,188 +115,334 @@ export const metrics = {
     units: 'B'
   }),
 
-  apm_requests: new ApmEventsRateClusterMetric({
+  apm_server_requests: new ApmEventsRateClusterMetric({
     field: 'beats_stats.metrics.apm-server.server.request.count',
-    title: i18n.translate('xpack.monitoring.metrics.apm.requestsTitle', {
-      defaultMessage: 'Requests'
+    title: i18n.translate('xpack.monitoring.metrics.apm.server.requestsTitle', {
+      defaultMessage: 'Requests Count Intake API'
     }),
-    label: i18n.translate('xpack.monitoring.metrics.apm.requests.requestedLabel', {
+    label: i18n.translate('xpack.monitoring.metrics.apm.requests.server.requestedLabel', {
       defaultMessage: 'Requested'
     }),
-    description: i18n.translate('xpack.monitoring.metrics.apm.requests.requestedDescription', {
+    description: i18n.translate('xpack.monitoring.metrics.apm.requests.server.requestedDescription', {
       defaultMessage: 'HTTP Requests received by server'
     })
   }),
+  apm_acm_requests: new ApmEventsRateClusterMetric({
+    field: 'beats_stats.metrics.apm-server.acm.request.count',
+    title: i18n.translate('xpack.monitoring.metrics.apm.acm.requestsTitle', {
+      defaultMessage: 'Requests Count Agent Configuration Management',
+    }),
+    label: i18n.translate('xpack.monitoring.metrics.apm.acm.requests.requestedLabel', {
```

Comment:
Not sure if important, but for consistency you might want to use use `acm.request` rather than `acm.requests`